EventFilter/SiPixelRawToDigi/plugins/SiPixelRawToDigi.h

```diff
@@ -10,15 +10,23 @@
 #include "FWCore/Framework/interface/stream/EDProducer.h"
```

Comment:
we've been asking since a while to not make header files for (concrete) plugins. Since this file is rather heavily modified already, please conform to having no header as well and merge the content here in SiPixelRawToDigi.cc